src/test/resources/json/swagger.json

```diff
@@ -730,37 +730,48 @@
         }
     },
     "definitions": {
-        "User": {
+        "Identified": {
```

Comment:
This is a contrived example for inheritance - I didn't want to mess w/ the model too much, so I just pulled out a single property (`"id"`) into a separate definition